<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qv2ray</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>680</width>
     <height>530</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="4,6" rowminimumheight="0,0,0,0,0,8">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,0,0,0">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>#Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>#Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restartButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>#Restart</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearlogButton">
        <property name="text">
         <string>#ClearLog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="prefrencesBtn">
        <property name="text">
         <string>#Preferences</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>999999</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>#Stopped</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>#HostList</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QListWidget" name="connectionListWidget">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::CopyAction</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="importConfigBtn">
        <property name="text">
         <string>#ImportConfig</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addConfigBtn">
        <property name="text">
         <string>#AddConfig</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delConfigBtn">
        <property name="text">
         <string>#RemoveConfig</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QTextBrowser" name="logText">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>#Log</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>#ConfigDetail</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>#Host</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="_hostLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>#Port</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="_portLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>#OutBoundType</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="_OutBoundTypeLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>#DetailInfo</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextBrowser" name="detailInfoTxt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>#EditConnection</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="editConnectionSettingsBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>#ConnectionSettings</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEdit">
   <property name="text">
    <string>#ManuallyCreateConnection</string>
   </property>
  </action>
  <action name="actionExisting_config">
   <property name="text">
    <string>#ImportConnection</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>#Exit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>#Preferences</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="text">
    <string>#Start</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="text">
    <string>#Stop</string>
   </property>
  </action>
  <action name="action_Restart">
   <property name="text">
    <string>#Restart</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>restartButton</tabstop>
  <tabstop>clearlogButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
